stories/__tests__/SQFormMultiSelect.stories.test.js

```diff
@@ -0,0 +1,142 @@
+    render(<SQFormMultiSelect size="auto" />);
+
+    const expandButton = await waitFor(() =>
+      screen.findByRole('button', {name: /friends/i})
+    );
+    await waitFor(() =>
+      expect(expandButton).toHaveTextContent(initialDropdownValue)
+    );
+  });
+
+  it('should show empty value "- -" if provided initial value not in options', async () => {
+    render(<SQFormMultiSelect size="auto" initialValue={{friends: [0]}} />);
+
+    const expandButton = await waitFor(() =>
+      screen.findByRole('button', {name: /friends/i})
+    );
+    await waitFor(() =>
+      expect(expandButton).toHaveTextContent(initialDropdownValue)
+    );
+  });
+
+  it('should show empty list if no options passed', async () => {
+    const consoleWarnSpy = jest
+      .spyOn(console, 'warn')
+      .mockImplementation(() => {});
+
+    render(
+      <SQFormMultiSelect useSelectAll={false} size="auto" children={null} />
+    );
+
+    const expandButton = await screen.findByRole('button', {name: /friends/i});
+
+    userEvent.click(expandButton);
+    expect(screen.queryByRole('option')).not.toBeInTheDocument();
+    expect(consoleWarnSpy).toHaveBeenCalledWith(
+      expect.stringMatching(/the children you provided.*was undefined/i)
+    );
+
+    consoleWarnSpy.mockRestore();
+  });
+
+  it('should update when options are selected', async () => {
+    render(<SQFormMultiSelect size="auto" />);
+
+    const expandButton = await screen.findByRole('button', {name: /friends/i});
+
+    expect(expandButton).toHaveTextContent('- -');
+
+    userEvent.click(expandButton);
+
+    const option1 = await screen.findByText('Jack');
+    userEvent.click(option1);
+
+    const option2 = await screen.findByText('Jill');
+    userEvent.click(option2);
+
+    await waitFor(() => expect(expandButton).toHaveTextContent('Jack, Jill'));
+  });
+
+  it('should not open dropdown options if field is disabled', async () => {
+    render(<SQFormMultiSelect size="auto" isDisabled={true} />);
+
+    const expandButton = await screen.findByRole('button', {name: /friends/i});
+    expect(expandButton).toHaveAttribute('aria-disabled');
+  });
+
+  it('should display icon and text if field is required', async () => {
+    render(<SQFormMultiSelect isRequired={true} size="auto" />);
+
+    const required = await screen.findByText(/required/i);
+    expect(required).toBeVisible();
+  });
+
+  it('should not display icon and text if field is not required', async () => {
+    render(<SQFormMultiSelect isRequired={false} size="auto" />);
+
+    const required = screen.queryByText(/required/i);
+    await waitFor(() => expect(required).not.toBeInTheDocument());
+  });
+
+  it('should highlight field if required but no value selected', async () => {
+    const validationSchema = {
+      friends: Yup.string().required('Required')
+    };
+
+    render(
+      <SQFormMultiSelect
+        size="auto"
+        isRequired
+        validationSchema={validationSchema}
```

Comment:
Same as my previous comment about `initialValues` here. `SQFormProps` needs to be exposed on the story to pass in a schema to `SQForm`